Added Type 10 calibration description (#186)
icub-tech-iit/documentation: docs/icub_r1_icub3_calibration_types/icub_r1_calibration_types.md

|`calibrationDelta`| defined by user expressed in degree|

### Calibration 10

> This calibration is used for joint with relative encoder, that is calibrated by moving against hard stop limit. It is used for example in R1 lower arm pronosupination and in iCub 3 lower body when AMO sensors are used as relative encoders. The AMO magnetic targer is subdivided into 64 or 32 sectors. The AMO sensor provides 14 bits resolution per sector, for a total 14+6=20 or 14+5=19 bits per revolution resolution respectively. The `Gearbox_E2J` parameter must be configured with the number of sectors in the corresponding `mechanicals/part-mec.xml` file.

Reviewer: ```suggestion
This calibration is used for joints with relative encoders, which are calibrated by moving against the hard-stop limit. It is used for example in R1 lower arm pronosupination and in iCub 3 lower body when AMO sensors are used as relative encoders.

Regarding iCub 3 and the AMO boards, the AMO magnetic target is subdivided into 64 or 32 sectors. The AMO sensor provides a 14-bit resolution per sector, for a total of 14+6=20 or 14+5=19 bits per revolution resolution, respectively. The `Gearbox_E2J` parameter must be configured with the number of sectors in the corresponding `mechanicals/part-mec.xml` file.
```
